go-controller/pkg/clustermanager/userdefinednetwork/controller_test.go

```diff
@@ -701,6 +701,52 @@ var _ = Describe("User Defined Network Controller", func() {
 					}
 				})
 			})
+
+			It("when started, CR exist, stale NADs exist, should deleted stale NADs", func() {
```

Comment:
where is the code that finds stale nads and cleans them?

Reply:
My comment was really about NADs that exist at start up that are owned by the CUDN controller, but they no longer have a corresponding CUDN. @npinaeva pointed out that isn't possible due to the finalizer, so I guess it's not something we need to worry about.